TST: missing seed and test failure in qsturng

try except leaves a variable undefined

test error in https://travis-ci.org/statsmodels/statsmodels/jobs/444183621

```
statsmodels/stats/libqsturng/qsturng_.py:731: in _qsturng
    r0_sq = _interpolate_p(p, r, v0)**2
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
p = 0.902896898398093, r = 9, v = 1
    def _interpolate_p(p, r, v):
        """
        interpolates p based on the values in the A table for the
        scalar value of r and the scalar value of v
        """
    
        # interpolate p (v should be in table)
        # if .5 < p < .75 use linear interpolation in q
        # if p > .75 use quadratic interpolation in log(y + r/v)
        # by -1. / (1. + 1.5 * _phi((1. + p)/2.))
    
        # find the 3 closest v values
        p0, p1, p2 = _select_ps(p)
        try:
            y0 = _func(A[(p0, v)], p0, r, v) + 1.
        except:
            print(p,r,v)
        y1 = _func(A[(p1, v)], p1, r, v) + 1.
        y2 = _func(A[(p2, v)], p2, r, v) + 1.
    
>       y_log0 = math.log(y0 + float(r)/float(v))
E       UnboundLocalError: local variable 'y0' referenced before assignment
```

--- a/statsmodels/stats/libqsturng/qsturng_.py
+++ b/statsmodels/stats/libqsturng/qsturng_.py
@@ -531,6 +531,7 @@ def _interpolate_p(p, r, v):
         y0 = _func(A[(p0, v)], p0, r, v) + 1.
     except:
         print(p,r,v)
+        raise
     y1 = _func(A[(p1, v)], p1, r, v) + 1.
     y2 = _func(A[(p2, v)], p2, r, v) + 1.
 

